src/layout/TagsInput.js

```diff
@@ -8,9 +8,16 @@ import { Flex } from './Flex'
 import Popover from './Popover'
 import OutsideClickHandler from 'react-outside-click-handler'
 
-let isValidTag = (tag, tags) => {
-  let cleanTag = _.trim(tag)
-  return !_.isEmpty(cleanTag) && !_.includes(cleanTag, tags)
+let isValidInput = (tag, tags) => !_.isEmpty(tag) && !_.includes(tag, tags)
+
+let getUniqueTagsFromInput = (tag, tags, splitCommas=true) => {
+  let trimmedTag = _.trim(tag)
+  if(splitCommas && _.includes(',', trimmedTag)) {
+    // Extract only the unique and not present in the current tags collection tags from the string
+    let newUniqTags = _.difference(_.uniq(_.compact(_.split(',', trimmedTag))), tags)
```

Comment:
`_.flow` was too long and had issues with `_.difference`- had to use `_.filter` etc so it ended up less concise